KVKCalendar/Classes/Timeline+Extension.swift

```diff
@@ -51,6 +51,7 @@ extension TimelineView {
             time.text = hour
             let formatter = DateFormatter()
             formatter.dateFormat = "HH:mm"
+            formatter.timeZone = style.timeline.currentLineHourTimeZone
```

Comment:
@kvyatkovskys while this changes the timelineLabel timezone, it affects the events time by placing them in the wrong boxes